README.md

```diff
@@ -1,7 +1,12 @@
-![librosa logo](docs/img/librosa_logo_text.svg)
+<div align="center">
+
+[![librosa logo](docs/img/librosa_logo_text.svg)](https://librosa.org/)
+
+# librosa
```

Comment:
i'd be in favor of removing this mention of "librosa", which looks like a duplicate of the logo and takes up unnecessary space.

Reply:
The first one is the logo (with a [hyperlink](https://librosa/org/) added if the user clicks on it)
And the second one is the Heading of the repository.

So I'm not really sure what the suggestion is here.